tests/test_multilabel_classification.py

```diff
@@ -304,6 +305,36 @@ def test_is_multilabel(labels):
     assert not ml_scorer._is_multilabel(labels[:, 0])
 
 
+def test_common_multilabel_issues():
+    pred_probs = np.array(
+        [
+            [0.9, 0.1, 0.0, 0.4, 0.1],
+            [0.7, 0.8, 0.2, 0.3, 0.1],
+            [0.9, 0.8, 0.4, 0.2, 0.1],
+            [0.1, 0.1, 0.8, 0.3, 0.1],
+            [0.4, 0.5, 0.1, 0.1, 0.1],
+            [0.1, 0.1, 0.2, 0.1, 0.1],
+            [0.8, 0.1, 0.2, 0.1, 0.1],
+        ]
+    )
+    labels = [[0], [0, 1], [0, 1], [2], [0, 2, 3], [], []]
+    df = common_multilabel_issues(labels=labels, pred_probs=pred_probs)
+    expected_issue_probabilities = [
+        0.14285714285714285,
+        0.14285714285714285,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+    ]
+    assert len(df) == 10
+    assert np.isclose(np.array(expected_issue_probabilities), df["Issue Probability"]).all()
+
```

Comment:
assert the other columns of this DF are properly named and also contain the proper values

Reply:
pass in `class_names` too and assert that it worked properly

Reply:
Added here: https://github.com/cleanlab/cleanlab/blob/3c1524496ec519d772cc322da637b80badfe764a/cleanlab/multilabel_classification/dataset.py#L52